partner_interest_group/models/res_partner_interest_groups.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class ResPartnerInterestGroup(models.Model):
+    _name = "res.partner.interest.groups"
```

Comment:
I suggest not to use plural form here as the object represent one interest group
```python
_name = "res.partner.interest.group"
```

Reply:
fixed